cypress/integration/ampSpec.js

```diff
@@ -18,18 +15,9 @@ describe('AMP Tests on a .amp page', () => {
   });
 
   it('should error gracefully', () => {
-    retrieve404BodyResponse(
-      '/news/articles/c85pqyj5m2ko.cake',
-      'No route was found for /news/articles/c85pqyj5m2ko.cake.',
-    );
-    retrieve404BodyResponse(
-      '/news/lol/c85pqyj5m2ko.amp',
-      'No route was found for /news/lol/c85pqyj5m2ko.amp.',
-    );
-    retrieve404BodyResponse(
-      '/cake/articles/c85pqyj5m2ko.amp',
-      'No route was found for /cake/articles/c85pqyj5m2ko.amp.',
-    );
+    testNonHTMLResponseCode('/news/articles/c85pqyj5m2ko.cake', 404);
```

Comment:
This method name reads weirdly to me.

```suggestion
    testResponseCode('/news/articles/c85pqyj5m2ko.cake', 404);
```

Reply:
I also felt this, but it wasn't a new function. What do you think, @jamesbrumpton?

Reply:
Agreed. It's not very explicit. Will look over this with @bcmn .

Reply:
Updated.